src/syscalls/deprecated_syscall_request.rs

```diff
@@ -138,6 +154,8 @@ pub(crate) struct DeprecatedDelegateCallRequest {
     pub(crate) calldata: Relocatable,
 }
 
+/// We are implementing a converter from different types to  DeprecatedSyscallRequest
```

Comment:
This doesn't sound right, `we are implementing` sounds like a work in progress. But this is already implemented. I would change this to something like:
```
/// Implementation of `From<DeprecatedEmitEventRequest>` for `DeprecatedSyscallRequest`.
```